hazyCalculator.js

```diff
@@ -36,7 +37,7 @@ module.exports.calculate = function (calculationSteps) {
 
   calculationSteps.forEach(nextCalculationStep => {
     if (!isAcceptableValue(nextCalculationStep)) {
```

Comment:
Looks like an extra ending bracket was added to the error string unintentionally.  That needs to be removed in the next commit.